contracts/LockedNORI.sol

```diff
@@ -530,7 +530,7 @@ contract LockedNORI is
       "lNORI: Recipient cannot be zero address"
     );
     require(
-      address(params.recipient) != _msgSender(),
+      !hasRole(TOKEN_GRANTER_ROLE, params.recipient),
```

Comment:
Should probably prevent assigning this role to anyone with a grant as well

Reply:
Hmm, yeah that's a good idea.